### PR TITLE
Day8lab

### PR DESCRIPTION
**Pull request by qlalberta**

### CODE REVIEW

File: sales_page/index.html [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
-            <input location = "location" type = "text"/>
-          <label>Avg cookies</label>
-            <input avgCookies = "avgCookies" type = "text"/>
-          <label>min Customer</label>
-            <input minCustomer = "minCustomer" type = "text"/>
-          <label>max Customer</label>
-            <input maxCustomer = "maxCustomer" type = "text"/>
+            <legend>Sales information</legend>
+            <lable>Location</label>
+            <input name = "location" type = "text"/>
```
Comment (traceyr) [inline sales_page/index.html:14]: You need some form validation for parts of your input form. This link is helpful! https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

File: sales_page/js/app.js [hunk 2]
```diff
@@ -130,6 +130,31 @@ seattleCenterSales.render();
 capitalHillSales.render();
 alkiSales.render();
 
+// generate form from input and append it to the table
+var salesForm = document.getElementById('addSalesForm');
+
+salesForm.addEventListener('submit', function(event) {
```
Comment (traceyr) [inline sales_page/js/app.js:136]: Great job getting the submit working!

File: sales_page/js/app.js [hunk 3]
```diff
@@ -130,6 +130,31 @@ seattleCenterSales.render();
...
+  event.preventDefault();
+  var location = event.target.location.value;
+  var avgCookie = event.target.avgCookie.value;
+  var minCustomer = event.target.minCustomer.value;
+  var maxCustomer = event.target.maxCustomer.value;
+  var newSales = new Sales(location, avgCookie, minCustomer, maxCustomer);
```
Comment (traceyr) [inline sales_page/js/app.js:142]: New constructor! Great job!

Comment (qlalberta) [reply sales_page/js/app.js:142]: Thx:)

File: sales_page/js/app.js (hunk 2 above)
Comment (qlalberta) [reply sales_page/js/app.js:136]: Thx!

File: sales_page/index.html (hunk 1 above)
Comment (qlalberta) [reply sales_page/index.html:14]: Do you mean to validate the input for the numbers are indeed numbers? I validate the location using javascript. Would you recommend all of them in the same html file? Please let me know. Thanks!  Also thanks for the link.

Comment (qlalberta) [reply sales_page/index.html:14]: Sorry, I think i misunderstood your point. Pls ignore my previous message.  I revised the code as suggested. Thanks.

**Pull request merged**